exonum-java-binding/tests_profile

```diff
@@ -49,8 +49,6 @@ export JAVA_LIB_DIR="$(find ${JAVA_HOME} -type f -name libjvm.\* | xargs -n1 dir
 echo "JAVA_LIB_DIR=${JAVA_LIB_DIR}"
 
 # Version of Rust used to build tests.
-## Stable works well unless you want benchmarks.
-# TODO: stable does not work well until ECR-1839 is resolved
```

Comment:
The bug is open. Also, there is https://jira.bf.local/browse/ECR-2982 :man_facepalming: